[17기 이지은] 크리에이터 어플라이 머지 요청

jenna8447 opened the pull request:
>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
creatorApply page 레이아웃 완료
		 
## 수정 사항들 자세한 내용
레이아웃 완료

## 기타 질문 및 특이 사항
첨부한 파일 데이터(input type="file)를 어떻게 전송하는지 잘 모르겠습니다.
첨부 이미지 미리보기 기능 구현을 어떤 식으로 해야하는지 궁금합니다. 

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [x] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [x] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [x] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [x] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.
- [x] Git rebase와 squash를 했고 커밋 수가 하나 인것을 확인 했습니다.

Tonyk0901 commented on src/Pages/CreatorApply/CreatorApply.js:
```diff
@@ -1,7 +1,333 @@
 
-function CreatorApply(props) {
-  return <div></div>;
+  handleValue = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
```
계산된 속성명 굿 입니다! 👍

Tonyk0901 commented on src/Pages/CreatorApply/CreatorApply.js:
```diff
@@ -1,7 +1,333 @@
+        gender: this.state.gender,
+        age: this.state.age,
+      }),
+    })
```
state 객체를 `fetch` 함수 호출 이전에 구조분해할당 하시면 더 깔끔하게 코드를 작성할 수 있겠네요! 👐

Tonyk0901 commented on src/styles/reset.scss:
```diff
@@ -0,0 +1,141 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
```
`GlobalStyles` 에서 이미 reset 을 하고 있기 때문에, `reset.scss`를 별도로 만들 필요는 없을 것 같습니다!

Tonyk0901 reviewed (commented):
고생 많으셨습니다 지은님! 😄

이미지 미리보기 기능 구현은 `URL.createObjectURL()` 를 사용해 볼 수 있을 것 같고,
업로드된 이미지 파일을 백엔드와 통신하는 것으로는 `FormData` 를 사용해 볼 수 있을 것 같습니다. (다만, 이 부분은 백엔드 팀과 어떻게 처리를 할지 이야기가 필요합니다!)

[createObjectURL](https://developer.mozilla.org/ko/docs/Web/API/URL/createObjectURL)
[formData](https://developer.mozilla.org/ko/docs/Web/API/FormData)
위 공식 문서를 참고해보세요!

일단은 키워드만 알려드리겠습니다 👐 어떻게 기능을 구현하는지는 구글링을 통해 금방 찾으실 수 있으실 겁니다!

한번 스스로 해보시고, 안되면 다시 한번 질문주세요! 😄 화이팅 입니다!

아참, 커밋 개수가 2개네요! 스쿼시 해주시고, 컨플릭 해결도 해주세요!

jenna8447 committed:
Add: creatorApply 레이아웃 완료

- ProductList, Product, CategoryModal, SortingModal, Filter, Checkbox, RadioBtn, Tag 컴포넌트 추가
- product, category, sorting  목데이터 추가
- config API 및 목데이터 주소 추가
- favicon 추가
- title명 수정
- category 데이터 수정
- config api 주소 수정
- 필터 및 정렬 함수 수정
- 멘토 리뷰 반영
- index.html conflict 해결
